extensions/amp-carousel/0.2/test/test-type-slides.js

```diff
@@ -280,15 +280,21 @@ describes.realWin(
         expect(eventSpy).to.have.not.been.called;
       });
 
-      it('should dispatch when changing slides', async () => {
-        const eventSpy = env.sandbox.spy();
-        container.addEventListener('slideChange', eventSpy);
+      it('should dispatch event with index and actionTrust when changing slides', async () => {
+        let event;
+        let counter = 0;
+        container.addEventListener('slideChange', (e) => {
+          event = e;
```

Comment:
suggestion: If you don't expect the counter to become greater than 1, we can add `expect(event).to.be.undefined` here and remove the extra variable.

Reply:
Good point. Done.